cmd/terramate/cli/cloud.go

```diff
@@ -538,13 +538,16 @@ func (c *cli) detectCloudMetadata() {
 	}
 
 	var prNumber int
+	var pushedAtFromEvent *time.Time
 	prFromEvent, err := tmgithub.GetEventPR()
 	if err != nil {
 		logger.Debug().Err(err).Msg("unable to get pull_request details from GITHUB_EVENT_PATH")
 	} else {
 		logger.Debug().Err(err).Msg("got pull_request details from GITHUB_EVENT_PATH")
 		c.cloud.run.prFromGHAEvent = prFromEvent
 		prNumber = prFromEvent.GetNumber()
+		pushedAtTimestamp := prFromEvent.GetHead().GetRepo().GetPushedAt()
```

Comment:
in this chaining, is it guaranteed that this path traversal will not panic if any of the intermediate fields do not exist?

Reply:
It was not safe, I just added nil checks.